kiosk-helm/values.yaml

```diff
@@ -0,0 +1,122 @@
+# Default values for kiosk.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+X11:
+  image:
+    repository: registry.suse.com/suse/kiosk/xorg
+    pullPolicy: Always
```

Comment:
do we want to default to `Always` ? this means on registry inaccessibilty the app is no longer working